subsys/net/lib/sntp/sntp_pkt.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2019 Intel Corporation
```

Comment:
I do not see any changes in this file, thus no copyright owner should be added IMHO.